shared-bindings/displayio/ColorConverter.c

```diff
@@ -43,24 +43,27 @@
 //|         """Create a ColorConverter object to convert color formats. Only supports RGB888 to RGB565
 //|         currently.
 //|         :param bool dither: Adds random noise to dither the output image"""
+//|         :param bool transparent_color: Sets one color in the colorset to transparent"""
 //|         ...
 //|
 
 // TODO(tannewt): Add support for other color formats.
 //|
 STATIC mp_obj_t displayio_colorconverter_make_new(const mp_obj_type_t *type, size_t n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
     enum { ARG_dither};
+    enum { ARG_transparent_color };
 
     static const mp_arg_t allowed_args[] = {
         { MP_QSTR_dither, MP_ARG_KW_ONLY | MP_ARG_BOOL, {.u_bool = false} },
+        { MP_QSTR_transparent_color, MP_ARG_KW_ONLY | MP_ARG_INT, {.uint32_t = 0} },
```

Comment:
This is where I run into issues because obviously this is not a `MP_ARG_INT` and `mp_arg_t` doesn't seem to have a type for `uint32_t`. I'd imagine the fact I am trying to do this and it doesn't exist is a bit of a code smell.

Reply:
```suggestion
        { MP_QSTR_transparent_color, MP_ARG_KW_ONLY | MP_ARG_INT, {.u_int = 0} },
```